app/ui/src/main/res/layout/account_list_item.xml

```diff
@@ -1,37 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:background="?android:attr/selectableItemBackground"
-    android:clickable="true"
-    android:focusable="true"
-    android:gravity="center_vertical"
-    android:minHeight="?android:attr/listPreferredItemHeightSmall"
-    android:orientation="vertical">
+              xmlns:tools="http://schemas.android.com/tools"
```

Comment:
We use 4 spaces for indentation in XML files. Please adjust your IDE's code formatting settings accordingly.

Reply:
I already use tabs. Android Studio's auto formatter just uses more spaces than originally used in this file